src/main/resources/db/migration/V6__Add_TestStatisticsView.sql

```diff
@@ -0,0 +1,27 @@
+-- 1. Создание функции для обновления представления
+CREATE OR REPLACE FUNCTION update_test_statistics_view()
+    RETURNS TRIGGER AS $$
+BEGIN
+    TRUNCATE TABLE test_statistics_view;
+
+    INSERT INTO test_statistics_view (user_id, set_id, test_count, average_score)
+    SELECT
+        user_id AS userId,
+        set_id AS setId,
+        COUNT(*) AS testCount,
+        COALESCE(AVG(correct_answers::decimal) * 100 / NULLIF(AVG(cards_number::decimal), 0), 0) AS averageScore
```

Comment:
`correct_answers` and `cards_number` might be `null`. We need to ignore such cases and return `averageScore` as `null`